src/aliceVision/system/hardwareContext.cpp

```diff
@@ -0,0 +1,61 @@
+#include "hardwareContext.hpp"
+
+#include "cpu.hpp"
+#include "MemoryInfo.hpp"
+#include <aliceVision/alicevision_omp.hpp>
+
+namespace aliceVision {
+
+void HardwareContext::setupFromCommandLine(boost::program_options::options_description & options)
```

Comment:
Thinking about it again, we should keep this function in CmdLine to avoid creating a dependency to program_option in all our core libraries (like aliceVision::feature, etc).